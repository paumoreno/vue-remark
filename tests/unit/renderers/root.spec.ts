import VueRemarkRoot from "@/renderers/root.vue";
import { mount } from "@vue/test-utils";

describe(VueRemarkRoot.name, () => {
  test("snapshot", () => {
    const wrapper = mount(VueRemarkRoot, {
      slots: {
        default: "something",
      },
    });

    expect(wrapper).toMatchSnapshot();
  });
});
