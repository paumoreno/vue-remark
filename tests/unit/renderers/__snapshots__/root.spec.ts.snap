// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VueRemarkRoot snapshot 1`] = `<div>something</div>`;
